RPIPs/RPIP-bounty_standards.md

```diff
@@ -0,0 +1,101 @@
+---
+rpip: #<to be assigned>
+title: Bounty Standards
+description: Describes standards and incentives for the production of effective bounty proposals.
+author: LongForWisdom (@LongForWisdom)
+discussions-to: <URL>
+status: Draft
+type: Meta
+created: 2024-01-04
+requires: 26
+---
+
+## Abstract
+
+TODO Write
+
+## Motivation
+
+TODO Write
+
+## Specification
+The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in RFC 2119. TODO Remove
+
+### Bounty Lifecycle
+![image](../assets/rpip-bounty_standards/bounty-lifecycle.png)
+
+#### Statuses
+**Draft - Proposed** - Bounty is described and proposed to the GMC for adoption. Bounty is not yet eligible for adoption.  
+**Draft - Defined** - Bounty is well defined and ready for completion by a bounty hunter with the relevant skillsets if adopted by the GMC. Bounty is now eligible for adoption.  
+**Draft - Rejected** - Bounty was rejected by the GMC at this time. Bounty is not eligible for completion.  
+**Open** - Bounty is adopted by the GMC for completion and listed on relevant frontends. Bounty is now eligible for completion.  
+**Closed - Completed** - A Bounty has been fully completed and paid out by the GMC. Bounty is no longer eligible for completion.  
+**Closed - Expired** - Bounty has met its expiration date. Bounty is no longer eligible for completion.  
+**Closed - Withdrawn** - GMC has withdrawn the bounty. Bounty is no longer eligible for completion.  
+
+### Bounty Incentive Levers
+Bounty Incentives Levers are OPTIONAL incentives that the pDAO empowers the GMC to use to incentivise the generation and successful completion of bounty proposals.
```

Comment:
Minor note: you've used British spelling here but American spelling of "incentivize" on line 79, I know we've previously used American spelling for proposals, I don't really mind which you go with as long as it's consistent.